extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -579,9 +588,10 @@ export const AzureProvisionDialog: React.FC = () => {
 
       // set review list
       const requireList = result.filter((item) => item.required);
-      setRequireResources(requireList);
       const optionalList = result.filter((item) => !item.required);
-      setEnabledResources(optionalList);
+      updateFormData('requiredResources', requireList);
```

Comment:
The list of required and optional resources are more like the list of subscriptions - they are what is possible to have selected.  Maybe they shouldn't be part of the form data like I said up top, but it can be done later.

Reply:
I'd like to defer this until later.